Out version of funcsigs is patched such that it works as necessary.

--- mock-3.0.5/setup.cfg
+++ mock-3.0.5/setup.cfg
@@ -31,7 +31,7 @@ keyword =
 [options]
 install_requires = 
 	six
-	funcsigs>=1;python_version<"3.3"
+	funcsigs>=0.4;python_version<"3.3"
 python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*
 packages = mock
 
--- mock-3.0.5/mock.egg-info/requires.txt
+++ mock-3.0.5/mock.egg-info/requires.txt
@@ -1,7 +1,7 @@
 six
 
 [:python_version < "3.3"]
-funcsigs>=1
+funcsigs>=0.4
 
 [build]
 twine
